Makefile

```diff
@@ -89,7 +89,10 @@ clippy-fix:
 	$(cargo) +$(nightly) clippy --fix -Z unstable-options --all-targets --allow-dirty --allow-staged
 
 install: tendermint
-	ANOMA_DEV=false $(cargo) install --path ./apps
+	$(cargo) build --release \
+		--manifest-path matchmaker/mm_token_exch/Cargo.toml && \
+	cp target/release/libmm_token_exch.{dll,dylib,so} ${HOME}/.cargo/bin
+	# ANOMA_DEV=false $(cargo) install --path ./apps
```

Comment:
Do you want install to install only the shared object for the matchmaker now?

Reply:
ah, no sorry